app/models/account.rb

```diff
@@ -11,8 +11,11 @@ def price_groups
   include DateHelper
 
   has_many   :account_users, :inverse_of => :account
-  has_one    :owner, :class_name => 'AccountUser', :conditions => { user_role: AccountUser::ACCOUNT_OWNER, deleted_at: nil }
-  has_one    :owner_user, :through => :owner, :source => :user
+  # Using a basic hash doesn't work with the `owner_user` :through association. It would
+  # only include the last item in the hash as part of the scoping. Consider changing
+  # when we get to Rails 4.
```

Comment:
Thanks for the explanation; it does feel like an AR bug. Maybe add a `TODO:` so `rake notes` will find it.